docs/installation.md

```diff
@@ -185,21 +186,42 @@ otellogs:
       changeowner:
         securityContext:
           privileged: true
-kube-prometheus-stack:
-  prometheus-node-exporter:
-    service:
-      port: 9200
-      targetPort: 9200
-  prometheusOperator:
-    namespaces:
-      additional:
-        - my-namespace
 tailing-sidecar-operator:
   scc:
     create: true
 ```
 
-so, it will look like the following:
+__Notice:__ Prometheus Operator is deployed by default on OpenShift platform,
+you may either limit scope for Prometheus Operator installed with Sumo Logic Kubernetes Collection using
+`kube-prometheus-stack.prometheusOperator.namespaces.additional` parameter in `user-values.yaml` or
+exclude namespaces for Prometheus Operator installed with Sumo Logic Kubernetes Collection
+using `kube-prometheus-stack.prometheusOperator.denyNamespaces` in `user-values.yaml`.
+For details see [Prometheus document](/docs/prometheus.md#prometheus-operator-in-the-cluster)
+
+### Setting cluster name
```

Comment:
Can we move the `Setting cluster name` section up, just below the `Required parameters` section?